gnosis/safe-react
# fix: Change mnemonic name based on argument

## What it solves
Resolves #2850

## How this PR fixes it
A side-effect was added to `useMnemonicName` to change the return value based on a argument change.

## How to test it
Open safe creation, go forward one step to `Name` step and note the safe name. Go back, change network and go forward. The name should have change.
